kubernetes/cache.go

```diff
@@ -41,18 +49,38 @@ type (
 		GetServices(namespace string) ([]core_v1.Service, error)
 		GetStatefulSet(namespace string, name string) (*apps_v1.StatefulSet, error)
 		GetStatefulSets(namespace string) ([]apps_v1.StatefulSet, error)
+		GetNamespaces() ([]core_v1.Namespace, error)
+
+		// Openshift caches
+		GetProjects() ([]osproject_v1.Project, error)
+
+		// Istio caches
+		GetVirtualServices(namespace string) (*GenericIstioObjectList, error)
+
+		// Allow subset of cache to be used
+		KubeCached(namespace string) bool
```

Comment:
Ok, this means that k8s objects for namespace NS are cached, right ?

Reply:
Yes.